docs/automation/vyos-govyos.rst

```diff
@@ -0,0 +1,203 @@
+:lastproofread: 2024-03-10
+
+.. _vyos-govyos:
+
+go-vyos
+=======
+
+go-vyos is a Go library designed for interacting with VyOS devices through
+their API. This documentation is intended to guide you in using go-vyos for
```

Comment:
```suggestion
their REST API. This documentation is intended to guide you in using go-vyos for
```

Reply:
Completely agree with that - I'm looking forward to the NETCONF API